core/kogitoservice/configmap.go

```diff
@@ -1,28 +1,12 @@
-// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
the header should be kept.

Reply:
headers are not added to these files because I had created these new files under new package `core` and I miss to add new package to scan under `addheader.sh` file. Now I had added headers in all missing files and update `addheader.sh` file.